### DISCUSSION TITLE
Build multiple crates for multiple targets?

### DISCUSSION THREAD

**Discussion by mdrssv** [corrosion-rs/corrosion#565] [General]
Hi,

is there a way to set `Rust_CARGO_TARGET` for an specific build rather than globally?

**Comment by jschwe** [comment]
There is `corrosion_set_hostbuild()` to compile a crate for the default host target instead of cross-compile target.
Since CMake doesn't support multiple targets in the same build (with the exception of macos x86 and aarch64), corrosion can't really support this in the general case.
